CHANGELOG.md

```diff
@@ -183,6 +183,7 @@
   execution failed][6918].
 - [Performance and readability of documentation panel was improved][6893]. The
   documentation is now split into separate pages, which are much smaller.
+- [Fixed cursor position when ctrl-clicking the node][7014].
```

Comment:
These are user-facing logs. Users reading them should understand what changed without the need to click something and without prior knowledge. Telling "fixed cursor position" does not really explain what changed to the reader. I'd be thankful If you could expand that a little bit.